### PR TITLE
feat: Add example application for instrumentWebframework

### PR DESCRIPTION
**Pull request by svetlanabrennan**
## Details
Converting our[ tutorial on instrumentWebFramework](https://github.com/newrelic/node-newrelic/blob/main/examples/shim/Webframework-Simple.md) into a sample app using a simple, custom web framework to show users how the API is used. 

Todos:

- [x] Fix trace name for api/users request
- [ ] Add screenshots to readme

### CODE REVIEW

File: custom-instrument-apps/instrument-webframework/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Example instrumentation of a web framework application
+This is an example app that uses the `newrelic.instrumentWebframework` [API function](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api/#instrumentWebframework) and associated [WebFramework shim API](https://newrelic.github.io/node-newrelic/WebFrameworkShim.html) to instrument a hypothetical web framework. This sample instrumentation is an similar implementation of the [New Relic web framework instrumentation tutorial](https://newrelic.github.io/node-newrelic/tutorial-Webframework-Simple.html).
```
Comment (jsumners-nr) [inline custom-instrument-apps/instrument-webframework/README.md:2]:
```suggestion
This is an example app that uses the `newrelic.instrumentWebframework` [API function](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api/#instrumentWebframework) and associated [WebFramework shim API](https://newrelic.github.io/node-newrelic/WebFrameworkShim.html) to instrument a hypothetical web framework. This sample instrumentation is similar to the [New Relic web framework instrumentation tutorial](https://newrelic.github.io/node-newrelic/tutorial-Webframework-Simple.html).
```

File: custom-instrument-apps/instrument-webframework/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+
+## Getting started
+
+1. Clone or fork, this repository,
```
Comment (jsumners-nr) [inline custom-instrument-apps/instrument-webframework/README.md:7]:
```suggestion
1. Clone or fork this repository
```

File: custom-instrument-apps/instrument-webframework/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+2. Go into this subdirectory,
```
Comment (jsumners-nr) [inline custom-instrument-apps/instrument-webframework/README.md:9]:
```suggestion
2. Navigate to the example's subdirectory:
```

File: custom-instrument-apps/instrument-webframework/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+``` sh
+cd newrelic-node-examples/custom-instrument-apps/instrument-webframework
+```
+
+3. Install dependencies and run application
```
Comment (jsumners-nr) [inline custom-instrument-apps/instrument-webframework/README.md:15]:
```suggestion
3. Install dependencies and run the application
```

File: custom-instrument-apps/instrument-webframework/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+```sh
+npm ci
+NEW_RELIC_LICENSE_KEY=<Your New Relic License Key> npm start
```
Comment (jsumners-nr) [inline custom-instrument-apps/instrument-webframework/README.md:19]: Personally, I would avoid suggesting that folks write their key to their shell history. It looks like the included `newrelic.js` has the same "replace me" string in it. So I'd insert a step before this one to prompt the reader to update this config first.

Comment (svetlanabrennan) [reply custom-instrument-apps/instrument-webframework/README.md]: I prefer putting the license key in the config file too. This has been updated.

**Review by jsumners-nr — APPROVED** [review]

**Pull request merged**